packages/rest/src/rest.server.ts

```diff
@@ -380,17 +386,11 @@ export class RestServer extends Context implements Server, HttpServerLike {
     return protocol + '://' + host;
   }
 
-  private async _redirectToSwaggerUI(
-    request: Request,
-    response: Response,
-    options: RestServerConfig,
-  ) {
+  private async _redirectToSwaggerUI(request: Request, response: Response) {
     const baseUrl =
-      options.apiExplorerUrl || 'https://loopback.io/api-explorer';
-    const openApiUrl = `${this._getUrlForClient(
-      request,
-      options,
-    )}/openapi.json`;
+      (this._config.apiExplorer && this._config.apiExplorer.explorerUrl) ||
```

Comment:
To avoid repetitive check for `this._config.apiExplorer`, I am proposing to always initialize this property in the constructor, so that it's either an empty object or the user-provider object. Then we can simplify this part as follows:

```ts
const baseUrl = this._config.apiExplorer.explorerUrl || 'https://loopback.io/api-explorer';
```

We can even set `explorerUrl` to the default value in the constructor, that would allow us to simplify this code even further.